src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/DefaultTagHelperFactsService.cs

```diff
@@ -209,6 +209,19 @@ public override IReadOnlyList<TagHelperDescriptor> GetTagHelpersGivenParent(TagH
             return stringifiedAttributes;
         }
 
+        internal override TagHelperBinding? GetNearestAncestorTagHelperBinding(IEnumerable<SyntaxNode> ancestors)
+        {
+            foreach (var ancestor in ancestors)
+            {
+                if ( ancestor is MarkupTagHelperElementSyntax tagHelperElement)
```

Comment:
```suggestion
                if (ancestor is MarkupTagHelperElementSyntax tagHelperElement)
```

Reply:
Some day auto-format on save will protect me from the fact that my spacebar is alternatingly too sensitive and not sensitive enough.

Reply:
Today is the day! (Though I haven't been brave enough to turn it on because I'm sure someone on Roslyn will yell at me!)
![image](https://user-images.githubusercontent.com/754264/154154905-f9cf358b-0f9a-4db1-a526-472c1a49818f.png)